Formula/coccinelle.rb

```diff
@@ -2,8 +2,8 @@ class Coccinelle < Formula
   desc "Program matching and transformation engine for C code"
   homepage "http://coccinelle.lip6.fr/"
   url "https://github.com/coccinelle/coccinelle.git",
-      tag:      "1.0.8",
-      revision: "d678c34afc0cfb479ad34f2225c57b1b8d3ebeae"
+      tag:      "1.1.0",
+      revision: "e84d3ddc7d4131b7e7e70c29d49eca09d35fabb6"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-only"
```
Only a small fraction of the source headers (and most of them seem to be bundled third-party software) specify GPLv2+ with the "at your option" statement, so I think this is strictly GPLv2.